src/command.js

```diff
@@ -78,4 +79,10 @@ export function addMatchImageSnapshotCommand(
     },
     matchImageSnapshotCommand(options)
   );
+
+  // After completing each test, we need to instruct the plugin to clean up
+  // the screenshots that were created.
+  afterEach(() => {
+    cy.task(CLEAN_SCREENSHOTS, null, { log: false });
```

Comment:
@NMinhNguyen [This should be correct](https://github.com/cypress-io/cypress/blob/55b352a11d0a664517ae054c67432d98abc2389e/packages/driver/src/cy/commands/task.coffee#L8-L17), and will ensure that implementation details of the approach aren't exposed to the user.